code/app/src/main/java/com/adobe/marketing/mobile/messagingsample/MessagingApplication.kt

```diff
@@ -32,17 +32,20 @@ class MessagingApplication : Application() {
         Edge.registerExtension();
 
         MobileCore.start {
-            // Necessary property id for MessagingSDKTest which has the edge configuration id needed by aep sdk
+            // Necessary property id which has the edge configuration id needed by aep sdk
             MobileCore.configureWithAppID("<appId>")
             MobileCore.lifecycleStart(null);
 
-            MobileCore.updateConfiguration(mutableMapOf("messaging.eventDataset" to "<expEventDatasetId>") as Map<String, Any>?)
+            // To configure the expereience event dataset for push tracking follow this url (https://aep-sdks.gitbook.io/docs/v/AEP-Edge-Docs/)
+            // Uncomment the below line if eventDataset is not configured in the messaging extension in launch
+            //MobileCore.updateConfiguration(mutableMapOf("messaging.eventDataset" to "<expEventDatasetId>") as Map<String, Any>?)
```

Comment:
Is there a common scenario where we need to manually configure `datasetID` manually ?

Reply:
Nope